### PR TITLE
fix: env fixes

### PR DESCRIPTION
**Pull request by Gaurabag**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by allendevops** [comment]
### Git leaks report

Description | File | Line | Secret
-------------|------|------|--------
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 18-18 | `ey****Nw`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 27-27 | `ey****PM`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 30-30 | `ey****TM`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 33-33 | `ey****Aw`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 340-341 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 358-359 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 381-382 | `"p****d"`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 65-65 | `ey****EI`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 76-76 | `ey****y4`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/alice.key | 1-51 | `--****--`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/bob.key | 1-51 | `--****--`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/carol.key | 1-51 | `--****--`

**Comment by allendevops** [comment]
### Sonar scan report

:white_check_mark: Sonar execution succeeded.

Exclusions: `**/.github/**,**/tests/**,**/*_test.go,*.yml,*.yaml,*.proto,Dockerfile,*.md,*.yaml,**/*.pb.go,**/*.pb.*.go,*.mod,*.json,*.out,Makefile,LICENSE,.gitignore,**/browser/**,**/*_interfaces.go,**/mock_*.go,*.html,*.xml,*.yml,*.yaml,*.json,*.out,*.py,**/mock*.go,**/_gen.go`

File: browser/flagr-ui/src/utils/apiUtil.js [hunk 1]
```diff
@@ -9,9 +9,45 @@ let flagrAxiosInstance;
...
+  
+	if (isLocalNetwork(hostname) || hostname.includes("dev") || hostname.includes("demo")) {
+	  return ENVS.DEV
+	}
+  
+	if (hostname.includes("stage")) {
+	  return ENVS.STAGE
+	}
+  
+	return ENVS.PROD
```
Comment (himanshu-allen) [inline browser/flagr-ui/src/utils/apiUtil.js:42]: default should not be prod, prod should be specific

File: browser/flagr-ui/src/constants.js [hunk 2]
```diff
@@ -4,9 +4,32 @@ const FLAGR_UI_POSSIBLE_ENTITY_TYPES = process.env.VUE_APP_FLAGR_UI_POSSIBLE_ENT
...
+
+const ENVS = {
+  PROD: 'PROD',
+  STAGE: 'STAGE',
+  DEV: 'DEV'
+}
+
+const ENVURLS = {
+  PROD: {
+    VUE_APP_API_URL : 'http://flagr.allen-live.in/api/v1',
```
Comment (himanshu-allen) [inline browser/flagr-ui/src/constants.js:16]: replace: **flagr --> flagr-new** for all api

File: browser/flagr-ui/src/utils/apiUtil.js [hunk 3]
```diff
@@ -9,9 +9,49 @@ let flagrAxiosInstance;
...
+		hostname.startsWith("10.0.") ||
+		hostname.startsWith("0.0.0.0") ||
+		hostname.endsWith(".local")
+	)
+}
+  
+export function determineEnv(hostname) {
+	if (!hostname) {
+	  console.warn("Couldn't determine host. Using production environment.")
+	  return ENVS.PROD
```
Comment (himanshu-allen) [inline browser/flagr-ui/src/utils/apiUtil.js:31]: this should be dev as well

**Review by himanshu-allen — APPROVED** [review]

**Pull request merged**